src/devices/bus/mc10/mcx128.cpp

```diff
@@ -368,4 +361,7 @@ alice_pak_mcx128_device::alice_pak_mcx128_device(const machine_config &mconfig,
 {
 }
 
+} // anonymous namespace
+
+DEFINE_DEVICE_TYPE_PRIVATE(MC10_PAK_MCX128, device_mc10cart_interface, mc10_pak_mcx128_device, "mc10_mcx128", "Darren Atkinson's MCX-128 cartridge")
 DEFINE_DEVICE_TYPE_PRIVATE(ALICE_PAK_MCX128, device_mc10cart_interface, alice_pak_mcx128_device, "alice_mcx128", "Darren Atkinson's MCX-128 cartridge (Alice ROM)")
```

Comment:
Do these actually need to be different device types at all?  Would making the ROMs BIOS options not be adequate?

Reply:
I am not the author of that part, I only applied to it the comment you made on private device and namespace to the neighbour file. But anyway, as I started to change it, I can push the change further.

If I understand the suggestion, it would be to have only one device type for mcx128, but add a configuration option on it to select between both ROMs described at the beginning of the file?

Reply:
If you look at a card like this: https://github.com/mamedev/mame/blob/master/src/devices/bus/a2bus/grappler.cpp

Note that the Grappler+ has options for the 3.0 and 3.2 ROM revisions.  You select them like `-sl1 grapplus,bios=v30` or `-sl1 grapplus,bios=v32`.

Reply:
I finally had time to look at this. I see a difference between the grappler and the mc10 situation. From what I understand, for grappler, both ROMs are valid for the card. So this is presented as an option, with a default value.

For the MCX128 device, you must use the ROM depending on the ROM installed for the particular flavour of the computer. Both computers are (almost) the same, but they have different system ROMs. And this, if you run the MC-10, you must use the ROM for MC-10 in the MCX128 device. And the same for Alice, with its own ROM.

So that's why the author originally made two devices I guess. Changing the ROM for the device does not represent a valid configuration.

Is there a way to have the "default ROM" depending on the emulated computer?

Reply:
Change done